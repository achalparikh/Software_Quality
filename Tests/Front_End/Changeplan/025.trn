10_admin_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
08_John_Doe_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
10_John_Doe_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _