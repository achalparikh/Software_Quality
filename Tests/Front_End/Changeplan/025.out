(not enough money message here)